README.md

```diff
@@ -25,6 +25,7 @@ A CLI tool and Go library for generating a Software Bill of Materials (SBOM) fro
 - Linux distribution identification
 - Works seamlessly with [Grype](https://github.com/anchore/grype) (a fast, modern vulnerability scanner)
 - Able to create signed SBOM attestations using the [in-toto specification](https://github.com/in-toto/attestation/blob/main/spec/README.md)
+- Convert between SBOM formats, such as CycloneDX (JSON, XML), SPDX, and Syft's own format.
```

Comment:
I think we can leave out the "(JSON, XML)" qualifier — there are multiple encoding formats for SPDX too, but we don't need to list them here in this summary IMHO